Assets/HoloToolkit/SpatialSound/Scripts/AudioInfluencerManager.cs

```diff
@@ -0,0 +1,242 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Class which supports IAudioInfluencers being used with audio sources.
+    /// </summary>
+    /// <remarks>
+    /// AudioInfluencerManager requires an AudioSource component. If one is not attached, it will be added automatically.
+    /// Each sound playing game object needs to have an AudioInfluencerManager attached in order to have it's audio influenced.
+    /// </remarks>
+    [DisallowMultipleComponent]
+    [RequireComponent(typeof(AudioSource))]
+    public class AudioInfluencerManager : MonoBehaviour
+    {
+        /// <summary>
+        /// Frequency below the nominal range of human hearing.
+        /// </summary>
+        /// <remarks>
+        /// This frequency can be used to set a high pass filter to allow all
+        /// audible frequencies through the filter.
+        /// </remarks>
+        public static readonly float NeutralLowFrequency = 10.0f;
+
+        /// <summary>
+        /// Frequency above the nominal range of human hearing.
+        /// </summary>
+        /// <remarks>
+        /// This frequency can be used to set a low pass filter to allow all
+        /// audible frequencies through the filter.
+        /// </remarks>
+        public static readonly float NeutralHighFrequency = 22000.0f;
+
+        /// <summary>
+        /// Time, in seconds, between audio influence updates.
+        /// </summary>
+        /// <remarks>
+        /// The UpdateInterval range is betweel 0.0 and 1.0, inclusive.
+        /// The default value is 0.25.
+        /// A value of 0.0f indicates that updates occur every frame.
+        /// </remarks>
+        [Tooltip("Time, in seconds, between audio influence updates.  0 indicates to update every frame.")]
+        [Range(0.0f, 1.0f)]
+        public float UpdateInterval = 0.25f;
```

Comment:
let's use encapsulation with a private serialized backing field.
(same for all public fields below)

Reply:
the values can currently be set in the inspector as a public field. marking them private and serialized will prevent scripts from modifying these values at runtime.

the intent is to allow both editor and runtime changes.

Reply:
Right, and if we use a public property to access them, then we still have both, as intended.